src/model/IntentAction.ts

```diff
@@ -1,17 +1,21 @@
 import BigNumber = require("bn.js");
 
 export class IntentAction {
-    to: string;
-    value: BigNumber;
-    data: string;
+    to: string
+    value: BigNumber
+    data: string
+    receive: string[]
 
     constructor(
         to: string,
         value: BigNumber,
-        data: string
+        data: string = "0x",
+        receive: string[] = []
```

Comment:
'receive' is not used in any feature, maybe we can add it later when we implement staticcalls and receipts.